src/Client/Client.h

```diff
@@ -29,7 +32,7 @@ class Client {
     const char * _port;
     Socket* _socket;
     std::string _clientID; //TODO inicializar con el XML? Tal vez el ID pueda ser el nombre de usuario, una vez que se haga el login utilizar username en este campo
-    Login* login;
+    Login* _login;
```

Comment:
Tratemos de no usar el _, no se cuando lo empezamos a usar pero en verdad preferiria que no (https://stackoverflow.com/questions/589535/naming-convention-for-public-and-private-variable)